timelock-impl/src/main/java/com/palantir/atlasdb/timelock/lock/LockLog.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.timelock.lock;
+
+import java.util.Collection;
+import java.util.UUID;
+
+import com.palantir.atlasdb.timelock.lock.LockEvents.RequestInfo;
+import com.palantir.atlasdb.util.AtlasDbMetrics;
+import com.palantir.lock.LockDescriptor;
+import com.palantir.lock.v2.LockRequestV2;
+import com.palantir.lock.v2.WaitForLocksRequest;
+
+public final class LockLog {
+
+    private static final long SLOW_LOCK_THRESHOLD_MILLIS = 1_000;
```

Comment:
It seems that our current implementation has a configurable slow lock threshold, and uses 10s as the default. Are we confident that 1s won't pack the logs?